Runtime/Integrations/Steam/SteamLobbyManager.cs

```diff
@@ -87,14 +89,14 @@ class LobbySearchBuilder : ILobbySearchBuilder {
       }
 
     public ILobbySearchBuilder Filter(string key, SearchComparison comparison, string value) {
-      var comp = (LobbyComparison)((int)comparison);
-      _query.AddStringFilter(key, value, comp);
+      //var comp = (LobbyComparison)((int)comparison);
+      //_query.AddStringFilter(key, value, comp);
       return this;
     }
 
     public ILobbySearchBuilder Filter(string key, SearchComparison comparison, int value) {
-      var comp = (LobbyComparison)((int)comparison);
-      _query.AddNumericalFilter(key, value, comp);
+      //var comp = (LobbyComparison)((int)comparison);
+      //_query.AddNumericalFilter(key, value, comp);
```

Comment:
Same here with this commented out section.

Reply:
Same comment as above. See the new merge (I'll push it in a bit)